app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :users
   belongs_to :posts
```

Comment:
- Lets create many to one relationship here.
`belongs_to :user`
`belongs_to :post`

Reply:
- [ ] I would recommend using singular for models, this is how Ruby on Rails handles the model naming.